src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_DAY_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (days < SHORT_TERM) {
+    return PRICE_PER_DAY * days;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
```

Comment:
Instead of combining the conditions, you can just firstly check if it's a long term, then if it's a short term. 
Just return the default price by default